src/Algebra/Construct/Add/Point.agda

```diff
@@ -0,0 +1,97 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Definition of algebraic structures we get from adding a disjoint point
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.Construct.Add.Point where
+
+open import Algebra.Bundles
+open import Algebra.Core using (Op₂)
+import Algebra.Definitions as Definitions
+import Algebra.Structures as Structures
+open import Data.Maybe.Relation.Binary.Pointwise
+open import Data.Product using (_,_)
+open import Level using (Level; _⊔_)
+open import Relation.Binary.Core
+open import Relation.Binary.Definitions
+import Relation.Binary.Structures as Rel
+open import Relation.Nullary.Construct.Add.Point
+
+private
+  variable
+    a ℓ : Level
+
+module _ {A : Set a} {_≈_ : Rel A ℓ}{refl-≈ : Reflexive _≈_} {op : Op₂ A} where
+  open Definitions {A = Pointed A} (Pointwise _≈_)
+  private
+    module Underlying = Definitions {A = A} _≈_
```

Comment:
I think if there are two different underlying equalities, probably more readable to just open `Definition` once unparameterised and provide the two equalities explicitly. You could give the second equality a meaningful name like `_≈∙_ = Pointwise _≈_`